cmd/utils/nodecmd/util.go

```diff
@@ -0,0 +1,237 @@
+// Modifications Copyright 2023 The klaytn Authors
+// Copyright 2016 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package nodecmd
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/klaytn/klaytn/accounts/keystore"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/cmd/utils"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/common/hexutil"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	"github.com/klaytn/klaytn/crypto"
+	"github.com/klaytn/klaytn/crypto/sha3"
+	"github.com/klaytn/klaytn/governance"
+	"github.com/klaytn/klaytn/params"
+	"github.com/klaytn/klaytn/rlp"
+	"gopkg.in/urfave/cli.v1"
+)
+
+const pgmInput = `
+<extra> <header file (json format)>
+<vote>  <bytes>
+<gov>   <bytes>
+<key>   <keystore path> <password>
+`
```

Comment:
This command (1) parses extra, vote, and gov; and (2) decodes keystore files.
I'm not sure if the subcommands are intuitive.
How about more explicit subcommand names like these?
```
ken util parse-extra <header file (json format)>
ken util parse-gov <bytes>
ken util parse-vote <bytes>
ken util decrypt-keystore <keystore path> <password>
```